linux_os/guide/system/software/integrity/crypto/configure_crypto_policy/oval/fedora.xml

```diff
@@ -0,0 +1,37 @@
+<def-group>
+  <definition class="compliance" id="configure_crypto_policy" version="1">
+    <metadata>
+      <title>Configure System Cryptographic Policies</title>
+      <affected family="unix">
+        <platform>multi_platform_fedora</platform>
+      </affected>
+      <description>Look for cipher policy correctly configured in /etc/crypto-policy/config.</description>
+    </metadata>
+    <criteria>
+      <criterion comment="check for cipher policy correctly configured in /etc/crypto-policy/config"
+      test_ref="test_configure_crypto_policy" />
```

Comment:
I think that time stamp of the config file needs to be checked as well. Editing the file manually without running `update-crypto-policies` causes a false-negative.

From comments in `/etc/crypto-policies/config`:
```
# After modifying this file, you need to run update-crypto-policies
# for the changes to propagate.
```

To check whether crypto-policy is applied, verify time stamp of files /etc/crypto-policies/state/current and /etc/crypto-policies/config. If timestamp of the former is newer than the latter, the policy is applied.

Reply:
@yuumasato yes... this PR does not fully implement everything. This will be done in a follow up PR.

Reply:
Fine.